lib/countwords.rb

```diff
@@ -0,0 +1,43 @@
+class CountWords
+
+  attr_accessor :word_count
```

Comment:
This `word_count` isn't used yet. You can probably delete all the `@@`s and then you would be using it.